sqlmesh/core/model/definition.py

```diff
@@ -1281,6 +1284,12 @@ def render_seed(self) -> t.Iterator[QueryOrDF]:
                 string_columns.append(name)
 
         for df in self._reader.read(batch_size=self.kind.batch_size):
+            for column in type_columns:
+                if column not in df:
+                    normalized_name = normalize_identifiers(column, dialect=self.dialect).name
+                    if normalized_name in df:
+                        df.rename(columns={normalized_name: column}, inplace=True)
```

Comment:
Should we do a single `rename` call here? We can do it in one go after the loop - only need to populate the dict within the loop.